cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -1,13 +1,32 @@
+import logging
 from typing import Any, Dict, List
 
+import requests
+
+from .settings import remote_setting_url
+
+logger = logging.getLogger(__name__)
+
 
 class RemoteSettings:
     recipes: List[Dict[str, Any]] = []
+    url: str = remote_setting_url
+
+    def get_recipes(self) -> List[Dict[str, Any]]:
+        return self.recipes
 
-    @classmethod
-    def get_recipes(cls) -> List[Dict[str, Any]]:
-        return cls.recipes
+    def update_recipes(self, new_recipes: List[Dict[str, Any]]) -> None:
+        self.recipes = new_recipes
 
-    @classmethod
-    def update_recipes(cls, new_recipes: List[Dict[str, Any]]) -> None:
-        cls.recipes = new_recipes
+    def fetch_recipes(self):
+        try:
+            response = requests.get(self.url)
+            response.raise_for_status()  # raises an HTTPError for bad status codes
+        except requests.exceptions.RequestException as e:
+            logger.error(f"Failed to fetch recipes: {e}")
+            raise e
+        if data := response.json().get("data", []):
```

Comment:
![50first-dates-jocko-the-walrus](https://github.com/mozilla/experimenter/assets/119884/0879ea6a-9944-4bb9-aac4-56be5d0b1f01)